docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2.1'
+services:
+  openmct:
+    image: "openmct:latest"
```

Comment:
You should use `build` section instead to build from Dockerfile directly.

Reply:
Agreed.